extensions/wikia/AdEngine3/src/templates/big-fancy-ad-above-config.js

```diff
@@ -9,6 +9,7 @@ const {
 } = universalAdPackage;
 
 export const getConfig = () => ({
+	isResolved: false,
```

Comment:
I'd avoid storing state in config. I know it's already happening but my feeling is that we should avoid it.
Please use another (not that good either but still a bit better) approach and rely on presence of class `theme-locked`, similarly to how `isSticky` variable is defined:
`const isResolved = container.classList.contains('theme-locked')`